packages/slate-react/src/constants/hotkeys.js

```diff
@@ -22,19 +22,19 @@ const ITALIC = isKeyHotkey('mod+i')
 
 const ENTER = isKeyHotkey('enter')
 const SHIFT_ENTER = isKeyHotkey('shift+enter')
-const SPLIT_BLOCK = e => ENTER(e) || SHIFT_ENTER(e)
+const SPLIT_BLOCK = e => !IS_IOS && (ENTER(e) || SHIFT_ENTER(e))
 
 const BACKSPACE = isKeyHotkey('backspace')
 const SHIFT_BACKSPACE = isKeyHotkey('shift+backspace')
 const DELETE = isKeyHotkey('delete')
 const SHIFT_DELETE = isKeyHotkey('shift+delete')
-const DELETE_BACKWARD = e => BACKSPACE(e) || SHIFT_BACKSPACE(e)
-const DELETE_FORWARD = e => DELETE(e) || SHIFT_DELETE(e)
+const DELETE_BACKWARD = e => !IS_IOS && (BACKSPACE(e) || SHIFT_BACKSPACE(e))
+const DELETE_FORWARD = e => !IS_IOS && (DELETE(e) || SHIFT_DELETE(e))
```

Comment:
What's the need for all these `!IS_IOS` additions here?

Reply:
Gotcha! That makes sense. If you're down to clean this one up, I'd like that. I think currently #1491 adds a few other unrelated style things that I'd rather not have combined with this bug fix.